packages/dev/core/src/ShadersWGSL/iblVoxelGrid.vertex.fx

```diff
@@ -48,14 +48,14 @@ fn convertToFloat(word: u32, byteInWord: u32, dataType: u32) -> f32 {
             return f32(value) / 255.0;
         }
         case 5122u: { // SHORT
-            let shift = (byteInWord / 2u) * 16u;
+            let shift = (byteInWord & 0xFFFFFFFEu) * 4u; // Align to 2-byte boundary

```

Comment:
I think it's 8u instead of 4u? If `byteInWord` equals 2, we want `shift` to be 16, not 8.

Reply:
Yes, you're right.